controllers/managedfusion_controller.go

```diff
@@ -693,8 +711,8 @@ func (r *ManagedFusionReconciler) reconcileMonitoringResources() error {
 	}
 
 	promRuleList := promv1.PrometheusRuleList{}
-	if err := r.list(&promRuleList); err != nil {
-		return fmt.Errorf("Could not list prometheus rules: %v", err)
+	if err := r.list(&promRuleList, client.InNamespace(r.Namespace)); err != nil {
```

Comment:
same as above

Reply:
I have reverted the changes to be moved to `datafoundation.go` at a future Pr